### ISSUE TITLE
General guidelines for importing genes

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/mondo-ingest#48]
1. https://github.com/monarch-initiative/mondo-ingest/issues/47
2. Gene associations must be confirmed by OMIM (!?, @nicolevasilevsky says clingen suggests d2gs)

**Comment by nicolevasilevsky** [comment]
- [ ] to do for myself: write a rough draft of guidelines. This disease is associated with gene. We will import genes if we can reasonably correspond to the monogenic disease. We will import all gene links that are connected to one disease and one disease only. Include enough details that the OMIM ingest can filter out the relationships, ie filter each case where there is more than one gene association (in this case, delete them all)

**Comment by sabrinatoro** [comment]
>We will import all gene links that are connected to one disease and one disease only.
I don't think we can do this: we have branches that are called "gene-related diseases", so you already have 1 gene - N diseases

**Comment by nicolevasilevsky** [comment]
I will need to look at this more closely but I'm not sure what you mean @sabrina. We have branches that are gene-related disease, and the children are 1 gene - 1 disease.

At any rate, I'll work on this and share the draft with you two for review. :)

**Comment by nicolevasilevsky** [comment]
@sabrinatoro I misunderstood your comment. You are right, sometimes we have single genes associated with more than one disease.

**Comment by nicolevasilevsky** [comment]
Draft:

1. We import genes from OMIM whenever there is only a single disease 2 gene association for a disease. We implicitly assume that these cases are monogenic diseases (which will be wrong in some cases, but will be mostly ok (according to our knowledge of OMIM)). We won't import disease 2 gene associations if there is more than one gene.
2. Genes can be associated with more than one disease, for example, APC is implicated in MONDO:0016613 'APC-related attenuated familial adenomatous polyposis' and MONDO:0021056 'familial adenomatous polyposis 1'
3. We may get disease to gene associations from external groups as well, such as ClinGen, who often request new gene-related disease terms.

@sabrinatoro @matentzn

**Comment by nicolevasilevsky** [comment]
discussed on curation call, need to discuss with technical group

**Comment by nicolevasilevsky** [comment]
revised text:

1. We slurp the gene association from OMIM whenever a disease has only 1 gene. We implicitly assume that these cases are monogenic diseases (which may not be fully optimal but will be mostly ok (according to our knowledge of OMIM)). We won't import disease to gene associations if there is more than one gene.
2. Genes can be associated with more than one disease, for example, APC is implicated in MONDO:0016613 'APC-related attenuated familial adenomatous polyposis' and MONDO:0021056 'familial adenomatous polyposis 1'
3. We may get disease to gene associations from external groups as well, such as ClinGen, who often request new gene-related disease terms.

@nicolevasilevsky add to Mondo documentation

**Comment by nicolevasilevsky** [comment]
assigning to @matentzn to work on updating the slurp pipeline
